docs/using-availability-zones.md

```diff
@@ -0,0 +1,220 @@
+# Availability Zones
+
+**Feature Status:** Alpha since v1.12.
+
+Kubernetes v1.12 adds support for [Azure availability zones (AZ)](https://azure.microsoft.com/en-us/global-infrastructure/availability-zones/). Nodes in availability zone will be added with label `failure-domain.beta.kubernetes.io/zone=<region>-<AZ>` and topology-aware provisioning is added for Azure managed disks storage class.
+
+**TOC:**
+
+<!-- TOC -->
+
+- [Pre-requirements](#pre-requirements)
+- [Node labels](#node-labels)
+- [Load Balancer](#load-balancer)
+- [Managed Disks](#managed-disks)
+    - [StorageClass examples](#storageclass-examples)
+    - [PV examples](#pv-examples)
+- [Appendix](#appendix)
+- [Reference](#reference)
+
+<!-- /TOC -->
+
+## Pre-requirements
+
+Because only standard load balancer is supported with AZ, it is a prerequisite to enable AZ for the cluster. It should be configured in Azure cloud provider configure file (e.g. `/etc/kubernetes/azure.json`):
+
+```json
+{
+    "loadBalancerSku": "standard",
+    ...
+}
+```
+
+If topology-aware provisioning feature is used, feature gates `VolumeScheduling` and `DynamicProvisioningScheduling` should be enabled on master components (e.g. kube-apiserver, kube-controller-manager and kube-scheduler).
```

Comment:
We've removed the DynamicProvisioningScheduling feature gate in 1.12.  So only VolumeScheduling is required.

Reply:
Yep, thanks for tip